docs/RESEARCH_METRICS.md

```
@@ -760,17 +771,14 @@ Recommend depositing in open repositories (OSF, Dataverse) for scientific transp
 
 ## Contact & Support
 
-**Technical Questions:**
-Open GitHub issue: https://github.com/yourusername/codoncanvas/issues
+**Technical Questions:** Open GitHub issue:
+https://github.com/yourusername/codoncanvas/issues
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)